cv/qc-cv.obo

```diff
@@ -391,6 +390,128 @@ is_a: QC:4000010 ! ID free
 is_a: QC:4000026 ! environment metric
 relationship: has_relation QC:4000013 ! QC metric relation: one run
 
+[Term]
+id: QC:4000069
+name: Fraction of peptide IDs around max MS1
+def: "Fraction of all peptides identified at least X min earlier or later than max MS1 for ID. The values are given in a corresponding list, where the first value is the X minutes before (negative) or after (positive) max MS1 and the second value is the fraction of IDs." [PSI:QC]
+comment: Estimates very early peak broadening.
+is_a: QC:4000005 ! corresponding lists
+is_a: QC:4000009 ! ID based
+is_a: QC:4000022 ! chromatogram metric
+relationship: has_relation QC:4000013 ! QC metric relation: one run
+
+[Term]
+id: QC:4000070
+name: Interquartile RT period for peptide identifications
+def: "The interquartile retention time period, in seconds, for all peptide identifications over the complete run." [PSI:QC]
```

Comment:
`interquartile` mean IRQ in the common sense, i.e. the RT of Q3-Q2 ?
Would it make sense to define this metric in relative terms, i.e. `(Q3-Q2) / (Q4-Q1)` where the denominator normalizes for the gradient length?! Otherwise a 2h gradient vs a 30min gradient will not be comparable (they might not even be after normalization, but its worth a try?)

Reply:
@mwalzer could you explain more?